exporter/prometheusexporter/go.mod

```diff
@@ -33,7 +33,6 @@ require (
 	github.com/aws/aws-sdk-go v1.42.48 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cenkalti/backoff/v4 v4.1.2 // indirect
-	github.com/census-instrumentation/opencensus-proto v0.3.0 // indirect
```

Comment:
How these appeared in the changeset?

Reply:
I've ran `make gotidy`... not sure what changes happen :))